tests/wakunode_rest/test_rest_filter.nim

```diff
@@ -36,51 +41,63 @@ proc testWakuNode(): WakuNode =
 
 
 type RestFilterTest = object
-  node1: WakuNode
-  node2: WakuNode
+  serviceNode: WakuNode
+  subscriberNode: WakuNode
   restServer: RestServerRef
+  restServerForService: RestServerRef
   messageCache: filter_api.MessageCache
   client: RestClientRef
+  clientTwdServiceNode: RestClientRef
 
 
 proc setupRestFilter(): Future[RestFilterTest] {.async.} =
-  result.node1 = testWakuNode()
-  result.node2 = testWakuNode()
+  result.serviceNode = testWakuNode()
```

Comment:
It is suggested to not use the implicit `result`. see https://status-im.github.io/nim-style-guide/errors.result.html